tests/test_core.py

```diff
@@ -2510,6 +2510,10 @@ def test_time_c(self):
   def test_gmtime(self):
     self.do_core_test('test_gmtime.c')
 
+  def test_gmtime_r(self):
+    # Confirm that gmtime_r does not leak when called in isolation.
+    self.do_core_test('test_gmtime_r.c')
```

Comment:
How about adding new tests like this to `other`? I'm not sure it's worthwhile running this in all opt modes, as the leak was a JS specific issue.

Reply:
Done.